components/SingleVcItem.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { useContext, useRef } from 'react';
```

Comment:
any specific reason, we are having separate file, previously, we combined the singlevcitem.tsx and vcitem.tsx.

Reply:
previously we have singlevcitem.tsx and vcitem.tsx to make autoselecting the vc if there is only one and I have kept it like that. tested removing the singlevcitem.tsx it is working fine even it is removed so we dont need singlevcitem.tsx now removing the file now.